ui/src/components/admin/announcements/AdminAnnouncementContainer.js

```diff
@@ -0,0 +1,391 @@
+        <Button colorScheme="blue" size="lg" onClick={onCreate}>
+          Add New
+        </Button>
+        <Button colorScheme="blue" size="lg" onClick={toggleAnnouncementsView}>
+          {isCurrentAnnouncements
+            ? 'Past Announcements'
+            : 'Current Announcements'}
+        </Button>
+      </Stack>
+      <Heading as="h2" size="lg" pt={7}>
+        {isCurrentAnnouncements
+          ? 'Current Announcements'
+          : 'Past Announcements'}
+      </Heading>
+      <List spacing="2" pt={3}>
+        {/* List announcements */}
+        {announcementList.map((announcementItem) => (
+          <ListItem key={announcementItem.id}>
+            <Box p="3" borderRadius="lg" borderWidth="1px">
+              <Flex direction={['column', 'row']} spacing={1}>
+                <Box maxW="12rem" pr={5}>
+                  <Image
+                    src={announcementItem.imageAdUrl}
+                    fallbackSrc="https://hongkong.sub.hmcc.net/wp-content/uploads/Screenshot-2020-09-04-at-6.39.50-PM.png"
+                  />
+                </Box>
+                <Stack direction="column" spacing={1}>
+                  <Heading size="md">{announcementItem.title}</Heading>
+                  <Grid templateColumns="repeat(2, 1fr)" gap={1}>
+                    <Text>
+                      <CalendarIcon /> Date:{' '}
+                      {showProperDate(
+                        announcementItem.startDate,
+                        announcementItem.endDate
+                      )}
+                    </Text>
+                    <Text>
+                      <TimeIcon /> Time:{' '}
+                      {showProperTime(
+                        announcementItem.startTime,
+                        announcementItem.endTime
+                      )}
+                    </Text>
+                    <Text>
+                      <InfoOutlineIcon /> Location:{' '}
+                      {announcementItem.location
+                        ? announcementItem.location
+                        : '-'}
+                    </Text>
+                    <Text>
+                      <ChatIcon /> Submitter: {announcementItem.submittedBy}
+                    </Text>
+                    <Text>
+                      <ViewIcon /> Announce in:{' '}
+                      {announcementItem.isInWeb && (
+                        <Badge colorScheme="teal">Web</Badge>
+                      )}{' '}
+                      &nbsp;
+                      {announcementItem.isInPpt && (
+                        <Badge colorScheme="orange">PPT</Badge>
+                      )}
+                    </Text>
+                  </Grid>
+                </Stack>
+                <Spacer />
+                {/* Buttons to publish, edit, duplicate, delete */}
+                <HStack spacing={1}>
+                  {announcementItem.isInWeb && isCurrentAnnouncements && (
+                    <Button
+                      colorScheme="purple"
```

Comment:
minor ux request: can we do a diff color for wether its published or not to more easily see

Reply:
ok i changed the bg color